config/prow/cluster/boskos_janitor.yaml

```diff
@@ -32,3 +32,40 @@ spec:
         - name: credentials
           secret:
             secretName: ibmcloud-janitor-secret
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: boskos-janitor-ibmcloud-k8s
+  labels:
+    app: boskos-janitor-ibmcloud-k8s
+  namespace: test-pods
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: boskos-janitor-ibmcloud-k8s
+  template:
+    metadata:
+      labels:
+        app: boskos-janitor-ibmcloud-k8s
+    spec:
+      terminationGracePeriodSeconds: 300
+      containers:
+      - name: boskos-janitor-ibmcloud
+        image: gcr.io/k8s-staging-boskos/ibmcloud-janitor-boskos:latest
+        args:
+        - --boskos-url=http://boskos.test-pods.svc.cluster.local.
```

Comment:
This will work for both deployments, only the ` - --resource-type` will be different, where it will use the boskos client to acquire the dirty resource and clean it up.